code/game/atoms.dm

```diff
@@ -1469,7 +1469,7 @@
 		T = get_turf(src)
 
 	if(!T)
-		return 0
+		return FALSE
```

Comment:
Wrong, gravity is a numeric value. The FALSE below is incorrect here.

Reply:
Bits below were false I'll fix those too